src/Styles.js

```diff
@@ -102,8 +100,9 @@ export default css`
   display: flex;
   flex-direction: row;
   align-items: center;
-  border-bottom: 1px var(--api-parameters-document-title-border-color, #e5e5e5) solid;
-  border: var(--api-parameters-document-title-border);
+  border-bottom: var(--api-parameters-document-title-border-bottom,
```

Comment:
This is more restrictive than what we have now. I would at least keep the `border` attribute, and use this current `border-bottom` value as the default `border-bottom` value for the `border` attribute

Reply:
@TwoPlusTwoOne but this is not how it works. `border` controls the border of all sides at once. I've already mentioned that and a possible solution in my more detailed explanation here:
https://github.com/advanced-rest-client/api-body-document/pull/26#discussion_r643718478

Reply:
Yes I understand, but why not do something like this instead:

```
border: var(--api-parameters-document-title-border);
border-bottom: var(--api-parameters-document-title-border-bottom, 1px var(--api-parameters-document-title-border-color, #e5e5e5) solid);
```

This would cover being able to change the`border-bottom` and override `border`, which would fit what you mentioned in that PR, correct?

Reply:
@TwoPlusTwoOne, @benri, any news regarding this change? And what about this one https://github.com/advanced-rest-client/api-body-document/pull/26?